object/types.proto

```diff
@@ -29,6 +29,9 @@ enum ObjectType {
 
   // Object lock
   LOCK = 3;
+
+  // Initialization part of the split regular object
+  INIT = 4;
```

Comment:
So, a regular object with regular meta, but just a different type, right?

Reply:
yes. we can discuss if there are any other worthwhile options. also, we can think if the link object differs from the suggested init object and if we should make it special too